pkg/doc/verifiable/presentation_test.go

```diff
@@ -354,22 +354,42 @@ func TestPresentation_SetCredentials(t *testing.T) {
 	r := require.New(t)
 	vp := Presentation{}
 
+	vc, err := NewUnverifiedCredential([]byte(validCredential))
```

Comment:
Did you try setting multiple VCs together?

Reply:
I tried it in `example_presentation_test.go`, but I will add the example here (in the unit test) as well, thanks!